### ISSUE TITLE
clearml-data close: Error: unsupported operand type(s) for +=: 'int' and 'NoneType'

### ISSUE THREAD

**Issue by TuMePJlaH** [allegroai/clearml#755]
I have dataset:
![image](https://user-images.githubusercontent.com/17141754/186868600-3daf2725-36df-4983-86af-0e5dcadc9b8d.png)

I did the next steps:
``` bash
clearml-data create --project Datasets --name HelloDataset
-> New dataset created id=c1b2bd2a4a524b1cbcf79a9c625673d2
clearml-data add --files HelloDataset/images_0/ --dataset-folder images_0
clearml-data close

clearml-data create --project Datasets --name HelloDataset --parents c1b2bd2a4a524b1cbcf79a9c625673d2
-> New dataset created id=8b61a90ad9c846f8bb92ce6d6228d59f
clearml-data add --files HelloDataset/images_1/ --dataset-folder images_1
clearml-data close

clearml-data create --project Datasets --name HelloDataset --parents 8b61a90ad9c846f8bb92ce6d6228d59f
-> New dataset created id=9a0b67d42d2f4a6d9a2ca87adf64abb2
clearml-data add --files HelloDataset/images_2/ --dataset-folder images_2
clearml-data close

clearml-data create --project Datasets --name HelloDataset --parents 9a0b67d42d2f4a6d9a2ca87adf64abb2
-> New dataset created id=8d64af5aff9340a88a203130d400d804
clearml-data remove --files images_0/000.jpg
clearml-data close

clearml-data create --project Datasets --name HelloDataset --parents 8d64af5aff9340a88a203130d400d804
clearml-data add --files HelloDataset/images_3/ --dataset-folder images_3
clearml-data close
-> Error: unsupported operand type(s) for +=: 'int' and 'NoneType'
```

## Environment
* Self hosted
* ClearML SDK Version 1.6.2 # get from python: `import clearml; clearml.__version__`
* ClearML Server: 1.6.0-213
* python 3.8.10
* linux 20.04

**Comment by jkhenning** [comment]
Hi @TuMePJlaH ,

I think this issue was resolves in ClearML SDK 1.6.4 - can you please see if it works for you?

**Comment by TuMePJlaH** [comment]
Hi @jkhenning,
Yep, 1.6.4 works fine!
Thanks for your help!
